### PR TITLE
PR : elkstack-4.2.6

### PR DESCRIPTION
**Pull request by Gazzonyx**
Puh-puh-pull-leeeease!
Contains two bug fixes and bumps the cookbook version.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '4.2.5'
+version          '4.2.6'
 
 depends 'apt'
 depends 'build-essential'
 depends 'chef-sugar'
 depends 'cron', '~> 1.4.3'
-depends 'elasticsearch', '~> 0.3'
+depends 'elasticsearch', '>= 0.3.7'
```
Comment (dschlenk) [inline metadata.rb:14]: I don't love modifying the pin to an aggressive minimum. Should be able to leave it as is so long as the version used is > 0.3 < 0.4

File: Berksfile [hunk 2]
```diff
@@ -1,16 +1,13 @@
-source "https://api.berkshelf.com"
+source "https://supermarket.chef.io"
 
 metadata
 
-# until https://github.com/elastic/cookbook-elasticsearch/pull/230
-cookbook 'elasticsearch', '~> 0.3', git:'git@github.com:racker/cookbook-elasticsearch.git'
+cookbook 'elasticsearch', git: 'https://github.com/rackspace-cookbooks/rackspace_elasticsearch.git'
+cookbook 'logstash', git: 'https://github.com/rackspace-cookbooks/rackspace_logstash.git'
```
Comment (dschlenk) [inline Berksfile]: whoops! I think I was wrong for pointing you at this ancient logstash cookbook repo. Looks like I had already switched back to lusis' and was pinned at a mythical 0.10 version that I can't find a tag for. Hash 3d7632612da325459242ea17ba02a2368a4e63a1.

**Comment by dschlenk** [comment]
@Gazzonyx close! I was talking about the pin in the metadata file.

**Comment by Gazzonyx** [comment]
Yeah.  I was trying to cherry pick commits and then merge the differences to keep the history, but update the metadata pin. I forgot that PRs update when I push even if I planned on rewriting the history of the thing with a rebase and then pushing.  I hate that "feature".  

Regardless, I stopped pushing since I can't get Berkshelf to successfully run an install.  Might as well wait until I'm done hacking it to pieces to send a full and correct PR to something that works.

**Pull request merged**